metricbeat/module/redis/info/info.go

```diff
@@ -172,27 +178,27 @@ func (m *MetricSeter) Setup(ms *helper.MetricSet) error {
 func (m *MetricSeter) Fetch(ms *helper.MetricSet) (events []common.MapStr, err error) {
 
 	for _, host := range ms.Config.Hosts {
-		c := m.redisPools[host].Get()
-
-		// TODO: Do better error reporting
-		if c == nil {
```

Comment:
Correct. Seems I was wrong there though. No need to check, per the `redigo` docs:

> // Get gets a connection. The application must close the returned connection.
> // This method always returns a valid connection so that applications can defer
> // error handling to the first use of the connection. If there is an error
> // getting an underlying connection, then the connection Err, Do, Send, Flush
> // and Receive methods return that error.
> func (p *Pool) Get() Conn {